src/ducks/modules/mock.js

```diff
@@ -7,23 +7,27 @@ import { nodeAttributesProperty } from './network';
 
 const MOCK_GENERATE_NODES = 'MOCK/GENERATE_NODES';
 
-const generateNodes = (howMany = 0) =>
+const keyForVarName = (vars, name) => {
+  const entry = Object.entries(vars).find(([, val]) => val.name === name);
+  return entry && entry[0];
+};
+
+const generateNodes = (variableDefs, typeKey, howMany = 0) =>
```

Comment:
I wonder if we need to use "real" names or we should move to populating with random words/numbers based on variable types in the registry? That way we could populate any node types?

Reply:
I'm fine with either approach. Since Architect includes a 'person' type by default and this is just a dev feature, I don't mind the hardcoded 'person'. But if you'd prefer the latter, I can update here or in a follow-up PR.

Reply:
Maybe having the dev feature generate nodes of various types would help us catch edge cases. For example hard coded text strings that don't make sense, or other faulty logic.

Reply:
This is now supported; implemented as @wwqrd described above. Mocks will be generated based on the current prompt's subject and apply any `additionalAttributes` defined on it.

Side note: `additionalAttributes` was being used to name the additional props for `addNodes`, which themselves contained the `additionalAttributes` defined in the protocol; I renamed the former to `additionalProperties` to avoid confusion. (This could affect the add/update actions, but I believe I caught all the usages.)